README.adoc

```diff
@@ -1,4 +1,4 @@
-= Address Book (Level 4)
+= PocketBook V1.1
```

Comment:
You might not want to put the V1.1 here. Instead use releases to let the user know what is the latest version.